### DISCUSSION TITLE
27CA9DF4-047D-49DC-93F7-74B7693AEAC8

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#163] [General]
# 27CA9DF4-047D-49DC-93F7-74B7693AEAC8

TOI-5205 is a red dwarf star located 284.5 light-years away from us. In 2022, a super Jupiter was discovered to be in an orbit of 1.63 days. When TOI-5205 b of Jupiter's mass passes in front of its host, it blocks approximately 7% of the light, making it one of the largest known exoplanet transits. Due to its massive size, this planet challenges the theory of planetary formation

https://celestia.mobi/addon?item=27CA9DF4-047D-49DC-93F7-74B7693AEAC8&lang=zh_CN

**Comment by fulangming** [comment]
Why is this star so purple

**Comment by fulangming** [comment]
What's wrong with this star?
